vars/checkPatch.groovy

```diff
@@ -51,8 +52,14 @@ def call(Map config = [:]) {
         ignored_files += ":" + config['ignored_files']
     }
 
+    def checkpatch_args="--notree --show-types -"
```

Comment:
What are these extra (new in this patch) arguments going to do/change in the checkpatch behaviour?

Reply:
They're the default options from code_review/jenkins_github_checkwarn.sh, so if this code sets anything, it needs to add those back in (and the ignore list which is why I don't like this approach).